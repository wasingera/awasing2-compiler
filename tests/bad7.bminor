'single quote strings don't exist'
"but normal ones do"
string: function() { return 'bad'; }
