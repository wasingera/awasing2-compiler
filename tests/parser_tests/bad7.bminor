f: function void () = {
    for (;;) {
        if x > 5 {
            print "asdf";
        }
    }
}
