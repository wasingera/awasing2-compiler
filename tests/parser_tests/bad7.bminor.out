parse error: syntax error
Parse failed: x
